imports/plugins/core/checkout/client/components/completedOrderItem.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Components, registerComponent } from "@reactioncommerce/reaction-components";
+
+const CompletedOrderItem = ({ item, handleDisplayMedia }) => {
+  let image;
+  if (handleDisplayMedia(item)) {
+    image = handleDisplayMedia(item).url();
+  } else {
+    image = "/resources/placeholder.gif";
```

Comment:
@rymorgan I think he means for the default **Product** image. Rather than using the one included with RC.